book/website/community-handbook/working-groups/working-groups-formalisation.md

```diff
@@ -0,0 +1,126 @@
+(ch-working-groups-formalising)=
+# Formalising a Working Group
+
+While any community member can join a Working Group (WG), setting up a WG involves a few extra steps to assess both the need and interest in its development, alongside sharing more information with the broader community. 
+This section is designed to guide the creation of WGs, from brainstorming to establishing structure.
+
+## Should this Working Group be Created?
+
+A few guiding questions that may help with the process of identifying if a WG should be created (or not!) are:
+
+* **Establishing Need**: What need does this WG fulfill for *The Turing Way* community?
+* **Establishing Requirements**: Does this topic or type of work require ongoing support, or can it be integrated into another format? (Should this be a WG, or can this task be accomplished in Collaboration Cafe?)
+* **Establishing Membership**: Who would be a member of this WG currently (at the time of founding)?
+
+### Establishing Purpose for the Working Group
+
+To create a WG, its purpose must be established. 
+This can be done independently, and/or with the assistance of delivery team staff and/or brainstorming with other community members and leaders. 
+Establishing the purpose of a WG may require a series of brainstorming sessions and workshops, with various resources available to facilitate this process.
+
+The *The Turing Way* Research Community Manager or Research Project Manager may also assist in this process through facilitated sessions conducted at a Collaboration Cafe community call or in dedicated meetings or asynchronous discussions.
+Please reach out to them if you have any questions about the process or would like support in getting started.
+
+A WG may focus on:
+1. A specific topic related to or of interest to *The Turing Way* community.
+   * For example: localisation, professionalising research infrastructure roles, accessibility in open source communities and outputs.
+2. A type of ongoing task or work currently being done within the project.
+   * For example: Book Dash event strategy and delivery, open-source infrastructure maintenance, providing training and mentorship.
+3. Any other aspect you would like to introduce to the community! `for discussion - is this true?`
+
+### Proposing a Working Group
+
+A short proposal should be created and shared as an issue on *The Turing Way*'s main GitHub repository. 
+This proposal doesn't have to be exhaustive but must outline the area of work being proposed and where needs or opportunities have been identified in the project, community, or within the broader open science or research ecosystem of *The Turing Way*. 
+Please identify and include at least 3 to 4 members who are interested in participating and contributing to the proposed area of work.
+
+Ideally, the proposing party should discuss their plan with *The Turing Way* Research Community Manager or invite support from other members of the project delivery team.
```

Comment:
Think we need to re-think this - honestly I don't see the need for an intermediary here and we have multiple project delivery teams now existing within the Turing Way, so I think we need to update this. 

My suggestions is: We should invite proposals to be posted on GitHub, invite community comments in Slack and via the newsletter, and community discussions at Collaboration Cafes and the Community Forums, with the Steering Committee ultimately giving the thumbs up. 

If the timeline is too short for set up (say someone wants to apply to an event as part of the Working Group with a deadline in the next month) then we can think about how to streamline approval or maybe give retroactive approval from the Steering Committee.